documentation/deployment/README.md

```diff
@@ -0,0 +1,217 @@
+
+You can now use the `ConfigParser` to configure your VPC and other components based on the settings provided in the `cdk.context.json` file. Here is an example of how to use it in a CDK stack.
+
+### Example VPC Stack
+
+```typescript
+import { App, Stack, StackProps } from 'aws-cdk-lib';
+import { OSMLVpc } from 'osml-cdk-constructs';
+import { ConfigParser } from './config-parser';
+
+export interface VpcStackProps extends StackProps {
+  readonly config: ConfigParser;
+}
+
+export class MyVpcStack extends Stack {
+  constructor(scope: App, id: string, props: VpcStackProps) {
+    super(scope, id, props);
+
+    new OSMLVpc(this, 'MyVPC', {
+      account: props.config.account,
+      config: props.config.vpcConfig
+    });
+  }
+}
+```
+
+## Custom Model Deployment Configuration Instructions
+
+Follow the steps below to configure and deploy your custom model using the provided configuration classes and AWS CDK constructs.
+
+### Update the `cdk.context.json` File
+
+Your `cdk.context.json` file contains the configuration details required for deploying your custom model. 
+You need to update this file with your specific configuration values. When configuring your custom model deployment, 
+you have the flexibility to either provide a URI for an existing container image or specify a build path to construct 
+the container from source. If you choose to supply a CONTAINER_URI, the deployment process will import the container
+directly from the specified URI, streamlining the setup by utilizing a pre-built image. Alternatively, if you opt to 
+build the container from source, you need to specify the CONTAINER_BUILD_PATH and optionally the CONTAINER_BUILD_TARGET. 
+This approach allows you to define a custom build process using a Dockerfile located at the build path, giving you the 
+ability to customize the container environment precisely to your requirements before deployment. This dual-option 
+configuration ensures that you can either leverage existing container images for rapid deployment or construct tailored 
+containers from source for more specialized needs. Below is an example of how to structure your `cdk.context.json` file:
+
+```json
+{
+  "projectName": "your-project-name",
+  "account": {
+    "accountId": "your-aws-account-id",
+    "region": "your-aws-region",
+    "prodLike": true
+  },
+  "customModelEndpoints": {
+    "deploy": true,
+    "modelName": "your-model-name",
+    "instance_type": "ml.m5.large",
+    "containerConfig": {
+      "CONTAINER_TAG": "your-container-tag",
+      "CONTAINER_URI": "your-container-uri",
+      "CONTAINER_BUILD_PATH": "your-build-path",
+      "CONTAINER_BUILD_TARGET": "your-build-target",
+      "CONTAINER_REPOSITORY": "your-repository-name",
+      "CONTAINER_DOCKERFILE": "Dockerfile"
+    },
+    "endpointConfig": {
+      "CONTAINER_ENV": {
+        "ENV_VAR1": "value1",
+        "ENV_VAR2": "value2"
+      }
+    },
+    "buildFromSource": true
```

Comment:
Would this be renamed to `BUILD_FROM_SOURCE`?

Reply:
Fixed